### ISSUE TITLE
Question about the equation?

### ISSUE THREAD

**Issue by dsj96** [gmftbyGMFTBY/Copyisallyouneed#4]
Nice work!
After read the paper. I am confused about the equation in 3.1 MODEL ARCHITECTURE section.
$H_i \in R^{i*dL}$ 
Does multiplying by Transformer layers $L$ have any special meaning? For CLM, input a prefix of length $i$ and output a matrix (hidden state) of $i * d$.
Do I misunderstood it? Thank you for your reply! : )

**Comment by gmftbyGMFTBY** [comment]
Thank you for your interest in our work. There is no special meaning about the transformer layers $L$. Actually, we set the prefix embedding as the last hidden state of the last token, which will be used to compute the logits about the next phrase or token.

**Comment by dsj96** [comment]
Thanks!
